stanford_paragraph_types.features.field_instance.inc

```diff
@@ -10,108 +10,140 @@
-        'views' => 'views',
+      'absolute_url' => 1,
+      'attributes' => array(
+        'class' => 'btn',
+        'configurable_class' => 0,
+        'configurable_title' => 1,
+        'rel' => '',
+        'target' => 'default',
+        'title' => '',
+      ),
+      'display' => array(
+        'url_cutoff' => 80,
       ),
+      'enable_tokens' => 1,
+      'rel_remove' => 'default',
+      'title' => 'optional',
+      'title_label_use_field_label' => 0,
+      'title_maxlength' => 50,
+      'title_value' => '',
+      'url' => 0,
       'user_register_form' => FALSE,
+      'validate_url' => 1,
     ),
     'widget' => array(
-      'active' => 1,
-      'module' => 'options',
+      'active' => 0,
+      'module' => 'link',
       'settings' => array(),
-      'type' => 'options_select',
-      'weight' => 1,
+      'type' => 'link_field',
+      'weight' => 3,
     ),
   );
 
   // Exported field_instance:
-  // 'paragraphs_item-stanford_gallery-field_s_image_gallery'.
-  $field_instances['paragraphs_item-stanford_gallery-field_s_image_gallery'] = array(
-    'bundle' => 'stanford_gallery',
+  // 'field_collection_item-field_p_cards_card-field_p_card_file'.
+  $field_instances['field_collection_item-field_p_cards_card-field_p_card_file'] = array(
+    'bundle' => 'field_p_cards_card',
     'deleted' => 0,
     'description' => '',
     'display' => array(
       'default' => array(
         'label' => 'hidden',
-        'module' => 'image',
-        'settings' => array(
-          'image_link' => 'file',
-          'image_style' => 'large-square',
-        ),
-        'type' => 'image',
-        'weight' => 0,
-      ),
-      'paragraphs_editor_preview' => array(
-        'label' => 'above',
+        'module' => 'file',
         'settings' => array(),
-        'type' => 'hidden',
-        'weight' => 0,
+        'type' => 'file_url_plain',
+        'weight' => 4,
       ),
     ),
-    'entity_type' => 'paragraphs_item',
-    'field_name' => 'field_s_image_gallery',
-    'label' => 'Image',
+    'entity_type' => 'field_collection_item',
+    'field_name' => 'field_p_card_file',
+    'label' => 'File',
     'required' => 0,
     'settings' => array(
-      'alt_field' => 0,
-      'default_image' => 0,
+      'description_field' => 1,
       'file_directory' => '',
-      'file_extensions' => 'png gif jpg jpeg',
+      'file_extensions' => 'txt pdf doc docx ppt pptx xls xlsx xml odt zip rtf',
       'max_filesize' => '',
-      'max_resolution' => '',
-      'min_resolution' => '',
-      'title_field' => 0,
       'user_register_form' => FALSE,
     ),
     'widget' => array(
       'active' => 1,
-      'module' => 'image',
+      'module' => 'file',
       'settings' => array(
         'insert' => 0,
         'insert_absolute' => 0,
         'insert_class' => '',
         'insert_default' => 'auto',
         'insert_styles' => array(
           'auto' => 'auto',
+          'colorbox__3-col-header' => 0,
```

Comment:
We don't rely on colorbox for anything in this module. Please remove these guys.